service/controller/catalog/resource/appcatalogentry/resource.go

```diff
@@ -74,15 +74,15 @@ func (r Resource) Name() string {
 	return Name
 }
 
-func (r *Resource) getCurrentEntryCRs(ctx context.Context, cr v1alpha1.AppCatalog) (map[string]*v1alpha1.AppCatalogEntry, error) {
-	r.logger.Debugf(ctx, "getting current appcatalogentries for appcatalog %#q", cr.Name)
+func (r *Resource) getCurrentEntryCRs(ctx context.Context, cr v1alpha1.Catalog) (map[string]*v1alpha1.AppCatalogEntry, error) {
+	r.logger.Debugf(ctx, "getting current appcatalogentries for catalog %#q", cr.Name)
 
 	currentEntryCRs := map[string]*v1alpha1.AppCatalogEntry{}
 
 	lo := metav1.ListOptions{
 		LabelSelector: fmt.Sprintf("%s=%s,%s=%s", label.ManagedBy, key.AppCatalogEntryManagedBy(project.Name()), label.CatalogName, cr.Name),
 	}
-	entries, err := r.k8sClient.G8sClient().ApplicationV1alpha1().AppCatalogEntries(metav1.NamespaceDefault).List(ctx, lo)
+	entries, err := r.k8sClient.G8sClient().ApplicationV1alpha1().AppCatalogEntries("").List(ctx, lo)
```

Comment:
To get all current AppCatalogEntries CRs across all namespaces, we should use the wildcard here.

Reply:
There is also `metav1.NamespaceAll` what about using that?

https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#pkg-constants